profiles/linkedin.py

```diff
@@ -39,7 +40,7 @@ def get_profiles(self, company, title, num_profiles):
     # LinkedIn Search is restricted to certain amount of people,
     # Can use google to search instead.
     def get_linkedin_urls_google_search(self, company, title):
```

Comment:
Move this to its own class in a separate file, this class should maintain state on which page its in. Because right now, it's limited to only getting 10 URLs, but what if we need more say 20? If we can this function again we will get the same 10 twice. Another problem is that what if of the 10 results this function returned, only 6 actually worked at where we wanted them to work? So the class have to save which page it's in, and offer an API to extract the results from  10 to 20.